extmod/machine_usb_device.c

```diff
@@ -157,6 +157,12 @@ static mp_obj_t usb_device_active(size_t n_args, const mp_obj_t *args) {
 }
 static MP_DEFINE_CONST_FUN_OBJ_VAR_BETWEEN(usb_device_active_obj, 1, 2, usb_device_active);
 
+static mp_obj_t usb_remote_wakeup(mp_obj_t self) {
+    tud_remote_wakeup();
```

Comment:
This functions return true if wake up succeeded (device was suspended, and host supports remote wakeup). Is there any value returning this to the caller, do you think?

Reply:
Yes please, as you say this should have minimal size impact and may be useful later if we add more suspend/resume awareness.